psalm.xml

```diff
@@ -26,6 +27,27 @@
                 <referencedMethod name="Laminas\Cache\Storage\Adapter\RedisClusterOptions::setResourceManager"/>
             </errorLevel>
         </InternalMethod>
+        <!--
+            Since these are usually suppressed anyway AND psalm is already thinking about removing this issue at all,
```

Comment:
IMO not a good idea: mixed assignments and their presence in the baseline highlights areas where we need to improve our code.